src/main/java/com/qaprosoft/carina/demo/guipractice/pages/ProductPage.java

```diff
@@ -0,0 +1,23 @@
+package com.qaprosoft.carina.demo.guipractice.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+
+public class ProductPage extends AbstractPage {
+
+	public ProductPage(WebDriver driver) {
+		super(driver);
+	}
+	
+	@FindBy(xpath = "//*[@id = 'tbodyid']//a")
+	private ExtendedWebElement addToCartButton;
```

Comment:
defined elements should be above methods, constructors

Reply:
Done!